Legal.md

```diff
@@ -56,13 +56,15 @@ There are only three steps needed to apply copyright and license properly in you
 
 Voila! You have now applied the copyright and license statements to your project.
 
-## Contributor License Agreement
+## Developer's Certificate Of Origin
 
 While the copyright for all work performed by if(we) employees on behalf of if(we) belongs to if(we) itself, the copyright for all work performed by external contributors belongs to each contributor. This is well and proper and the right way of things, but it can cause some administrative headaches.
 
 For instance, what if if(we) actually _were_ to create a non-profit foundation for supporting and administering all of its open source projects? In order to reassign these projects to the purview of the foundation, if(we) would need to receive approval from each and every copyright holder for each and every project. As each contributor holds copyright over their own work, this could mean dozens or even hundreds of copyright holders…for **each** project. Considering how often people move around, change email addresses, or otherwise go AWOL, tracking them all down in order to get their approval to transfer the project to the foundation would be a logistical and administrative nightmare.
 
-Enter the [Contributor License Agreement](./starter_box/CLA.md) (aka the CLA).
+Enter the [Developer's Certificate Of Origin](./starter_box/DCOO.md) (aka the DCOO).
+
+NOTE: ALL the below about the CLA needs to be replaced with information on the DCOO
```

Comment:
The basic idea is that:

1. We don't want people to contribute code to us that is not their own and which is already under some other copyright.

2. The DCO asks contributors to certify that they are actually the authors and/or allowed to contribute the code to the project.

The specifics are covered within the DCO itself.